foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_9_0/MLFlowDockerfileMigration.java

```diff
@@ -68,10 +69,22 @@ protected boolean performMigration(File file) {
         boolean isMigrated = false;
 
         try {
-            for (Map.Entry<String, String> entry : DOCKER_IMAGE_REFERENCE_MAP.entrySet()) {
-                replaceLiteralInFile(file, entry.getKey(), entry.getValue());
-                isMigrated = true;
+            // update FROM to point to community docker image:
```

Comment:
I: I also updated this to remove the `ARG` lines rather than turning them into empty ones. This leaves things a bit cleaner, IMO.